packages/webapp/src/components/Form/ContextForm/index.tsx

```diff
@@ -108,10 +107,6 @@ export const ContextForm = ({
       setIsEditing={setIsEditing}
       showCancelFlow={showCancelFlow}
       setShowCancelFlow={setShowCancelFlow}
-      onSave={(...args: any) => {
-        setActiveStepIndex(0);
```

Comment:
I found this has changed behaviour of the other form (it displays the first page of the form for a second). I believe it's working without it, but if any problems are found, we will need another solution.

https://github.com/user-attachments/assets/f10a38c1-b5e1-418b-958c-27a457b4e79d